packages/react/src/components/Forms/FileUploader/utils.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * FROM https://gist.github.com/zentala/1e6f72438796d74531803cc3833c039c
+ * @param bytes
+ * @param decimals
+ */
+export function formatBytes(bytes: number, decimals: number = 2) {
+  if (bytes === 0) return "0 Bytes";
+  const k = 1024;
+  const sizes = ["Bytes", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"];
+  const i = Math.floor(Math.log(bytes) / Math.log(k));
+  return parseFloat((bytes / k ** i).toFixed(decimals)) + " " + sizes[i];
```

Comment:
Just in order to improve understandability : 

```suggestion
  const units = ["Bytes", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"];
  const exponent = Math.floor(Math.log(bytes) / Math.log(k));
  return parseFloat((bytes / k ** i).toFixed(decimals)) + " " + units[exponent];
```